lib/fix-google-html.js

```diff
@@ -255,6 +255,28 @@ export function unwrapLineBreaks (node) {
   node.children = children;
 }
 
+/**
+ * Moves linebreaks outside of anchor elements,
+ * if the linebreak is the first and/or last child of the anchor.
+ * @param {RehypeNode} node 
+ */
+export function moveLinebreaksOutsideOfAnchors (node) {
+  visit(node, 'element', (node, index, parent) => {
+    console.log(node)
+    if (node.tagName === 'a') {
```

Comment:
You can limit to `<a>` elements by using:

```js
visit(node, n => n.tagName === 'a', (node, index, parent) => {
  // Code to deal with `<a>` elements here.
});
```

Or, if useful, some of these checks are defined and given a name at the top of the file: https://github.com/Mr0grog/google-docs-to-markdown/blob/a2690fe56b41d17dac16a8a686a29ac0975c2179/lib/fix-google-html.js#L67-L71

So you can do:

```js
// At top of file:
const isAnchor = (node) -> node.tagName === 'a';

// In your function:
visit(node, isAnchor, (node, index, parent) => {
  // Code to deal with `<a>` elements here.
});
```

---

Also please clean up the `console.log` calls!